platform.js

```diff
@@ -351,5 +361,19 @@ module.exports = {
     }
 
     return module.exports.spawn(command);
+  },
+  getNetworkIP() {
```

Comment:
@ahmedalsudani We need a way of knowing which IP address will be used to fetch remote content, even if there is more than one interface. This is the simplest method (it's the same one we're currently using on the Java player); if you have an alternative, let me know.